src/pages/workspace/accounting/qbo/export/QuickbooksPreferredExporterConfigurationPage.tsx

```diff
@@ -45,46 +44,52 @@ function QuickbooksPreferredExporterConfigurationPage({policy}: WithPolicyConnec
+            if (row.value !== qboConfig?.export?.exporter) {
+                Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.QBO, CONST.QUICKBOOKS_CONFIG.EXPORT, {exporter: row.value});
             }
             Navigation.goBack(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_PREFERRED_EXPORTER.getRoute(policyID));
         },
-        [policyID, exportConfiguration],
+        [qboConfig?.export?.exporter, policyID],
+    );
+
+    const headerContent = useMemo(
+        () => (
+            <>
+                <Text style={[styles.ph5, styles.pb5]}>{translate('workspace.accounting.exportPreferredExporterNote')}</Text>
+                <Text style={[styles.ph5, styles.pb5]}>{translate('workspace.accounting.exportPreferredExporterSubNote')}</Text>
+            </>
+        ),
+        [translate, styles.ph5, styles.pb5],
     );
 
     return (
-        <AccessOrNotFoundWrapper
+        <SelectionScreen
             policyID={policyID}
             accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN]}
             featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
-        >
-            <ScreenWrapper testID={QuickbooksPreferredExporterConfigurationPage.displayName}>
-                <HeaderWithBackButton title={translate('workspace.accounting.preferredExporter')} />
-                <SelectionList
-                    headerContent={
-                        <>
-                            <Text style={[styles.ph5, styles.pb5]}>{translate('workspace.accounting.exportPreferredExporterNote')}</Text>
-                            <Text style={[styles.ph5, styles.pb5]}>{translate('workspace.accounting.exportPreferredExporterSubNote')}</Text>
-                        </>
-                    }
-                    sections={[{data}]}
-                    ListItem={RadioListItem}
-                    onSelectRow={selectExporter}
-                    shouldDebounceRowSelect
-                    initiallyFocusedOptionKey={data.find((mode) => mode.isSelected)?.keyForList}
-                />
-            </ScreenWrapper>
-        </AccessOrNotFoundWrapper>
+            displayName={QuickbooksPreferredExporterConfigurationPage.displayName}
+            sections={[{data}]}
+            listItem={RadioListItem}
+            headerContent={headerContent}
+            onBackButtonPress={() => Navigation.goBack(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_EXPORT.getRoute(policyID))}
+            onSelectRow={selectExporter}
+            initiallyFocusedOptionKey={data.find((mode) => mode.isSelected)?.keyForList}
+            title="workspace.accounting.preferredExporter"
+            connectionName={CONST.POLICY.CONNECTIONS.NAME.QBO}
+            pendingAction={PolicyUtils.settingsPendingAction([CONST.QUICKBOOKS_CONFIG.EXPORTER], qboConfig?.pendingFields)}
+            errors={ErrorUtils.getLatestErrorField(qboConfig ?? {}, CONST.QUICKBOOKS_CONFIG.EXPORTER)}
```

Comment:
```suggestion
            errors={ErrorUtils.getLatestErrorField(qboConfig, CONST.QUICKBOOKS_CONFIG.EXPORTER)}
```

let's move it to the declaration